src/Test.Utilities.Sarif/FileDiffingUnitTests.cs

```diff
@@ -285,7 +308,10 @@ protected virtual string ConstructTestOutputFromInputResource(string inputResour
                 sb.AppendLine(
                     "To rebaseline with current behavior:");
 
-                string testDirectory = Path.Combine(ProductTestDataDirectory, "ExpectedOutputs");
+                // TODO I suspect this baseline command works in some contexts (when we have a specific type under
```

Comment:
>TODO [](http://example.com/codeflow?start=19&length=4)

do we add name with todo item

Reply:
Thanks! I actually think I located and fixed the bug, there was a suite in this repo that showed the potential issue (related to having an intermediate directory storing test files).